pyproject.toml

```diff
@@ -64,7 +64,7 @@ afterray = ["psutil", "setproctitle"]
 "doc/*" = ["I"]
 "python/ray/__init__.py" = ["I"]
 "python/ray/dag/__init__.py" = ["I"]
-"python/ray/util/*" = ["I"]
+"python/ray/air/__init__.py" = ["I"]
 "rllib/*" = ["I"]
```

Comment:
Can this be removed now?

Reply:
@czgdp1807 I merged this PR. Could you do a follow-up to see what inside pyproject.toml can be cleaned up?